packages/adapter/addon/rest.js

```diff
@@ -301,10 +301,10 @@ const RESTAdapter = Adapter.extend(BuildURLMixin, {
       if (this._fastboot) {
         return this._fastboot;
       }
-      return getOwner(this).lookup('service:fastboot');
+      return (this._fastboot = getOwner(this).lookup('service:fastboot'));
```

Comment:
This seems like a bugfix in a test/ci fix? Is this fixing a bug? Can we separate it into a different PR then?

Reply:
see the updated commit, I'll change the PR to also state bugfix, the bug is uncovered by the tests

Reply:
Don't see the bug in the commit, maybe you didn't push?

Reply:
I meant I added `BUGFIX` to the commit, I will also update the description with the nature of the bugfix.